### DISCUSSION TITLE
How to put stepper's step bar at the bottom of page

### DISCUSSION THREAD

**Discussion by yongtao-wang** [mantinedev/mantine#6140] [Q&A]
Hi,
I'm using the Stepper component, which works fine. Is there a way to put the step bar at the bottom of the page (or configure the position freely)? Thanks.

**Comment by yverby** [comment]
You can use Styles API
https://mantine.dev/core/stepper/#styles-api

```jsx
<Stepper classNames={{ steps: 'your-class-name' }}>
  ...
</Stepper>
```
